proposals/allow_time_based_window_for_policy_enforcement_behavior

```diff
@@ -0,0 +1,145 @@
+# Kyverno Design Proposal - Allow time-based window for policy enforcement behavior
+
+Author: [Prateek Nandle](https://github.com/Prateeknandle)
+
+## Contents
+
+- [Kyverno Design Proposal - Allow time-based window for policy enforcement behavior](#kyverno-design-proposal---allow-time-based-window-for-policy-enforcement-behaviour)
+  * [Overview](#overview)
+  * [Proposal](#proposal)
+  * [Goals](#goals)
+  * [Example](#example)
+  * [Implementation](#implementation)
+    + [Change in Policy Behaviour]
+    + [Activity Status Report]
+
+## Overview
+
+I want to propose a new feature for policies, which will allow Time-based window policy enforcement behavior. This new feature would help to change the activity of the policies, for a particular defined Time window, eg : not allowing config changes during the weekends. Also a report will be created to see which policies are active at present.
+
+## Proposal 
+
+Currently, there is no such concept which will allow the user/ admin to make a time window where a policy’s activity would be changed automatically. Once a policy is defined then only manually it could be changed.
+
+Changing manually is a tedious job, for eg, if an admin wants that no developer should change config file during a time slot, let's say weekend, then the admin needs to manually change the policy. This is tedious, because every weekend the admin needs to change the policy manually.
+
+After this proposal the problem would be solved, a new tag would be added in the policy named, so that within a specified time window the policy would be inactive and outside the window it’ll be active.
+
+## Goals
```

Comment:
3. To allow testing time-based policies using the Kyverno CLI.

Reply:
done

Reply:
Yes.